plugins/vendor/entities/entities/ix_vendor.lua

```diff
@@ -50,8 +50,6 @@ function ENT:InitPhysObj()
 	local bPhysObjCreated = self:PhysicsInitBox(mins, maxs)
 
 	if (bPhysObjCreated) then
-		self:SetMoveType(MOVETYPE_NONE)
```

Comment:
I don't know why, but if you use `SetMoveType` bounding box is being changed to one showed in first video